sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_transport_async.py

```diff
@@ -248,8 +249,8 @@ def __init__(
         self.raise_on_initial_eintr = raise_on_initial_eintr
         self._read_buffer = BytesIO()
         self.host, self.port = to_host_port(host, port)
-
-        self.connect_timeout = connect_timeout
+        # not used by asyncio connections
```

Comment:
What does this mean?

Reply:
We don't use `self.connect_timeout` on the async side at all when we create a connection using `asyncio.create_connection`. On the async side, this timeout value is only used by the WebsocketTransport, and not the SSLTransport.

Reply:
removing socket_timeout from the async transport attributes altogether, since they're not used at the transport level. They're only used at the connection level, where the timeout is passed into asyncio.wait_for()